lib/hexpm/web/templates/sitemap/docs_sitemap.xml.eex

```diff
@@ -5,7 +5,7 @@
     xsi:schemaLocation="http://www.sitemaps.org/schemas/sitemap/0.9 http://www.sitemaps.org/schemas/sitemap/0.9/sitemap.xsd">
   <%= for {name, docs_updated_at} <- @packages do %>
     <url>
-      <loc>https://hexdocs.pm/<%= name %>/</loc>
+      <loc><%= raw Hexpm.Utils.docs_url([name]) %></loc>
```

Comment:
The new function name is `Hexpm.Utils.docs_html_url`, make sure to rebase to latest master.